cuegui/cuegui/ServiceDialog.py

```diff
@@ -262,14 +259,16 @@ def refresh(self):
                 self.__services = opencue.api.getDefaultServices()
             else:
                 self.__services = self.__show.getServiceOverrides()
-        except Exception:
-            return
+        except Exception as e:
+            raise Exception(e)
 
         for service in self.__services:
-            item = QtWidgets.QListWidgetItem(service.name())
+            item = QtWidgets.QListWidgetItem(service.data.name)
             self.__service_list.addItem(item)
-            if service.name() in selected:
-                item.setSelected(True)
+
+            if selected:
```

Comment:
Do we need this `if`? If `selected` is an empty list that should be handled by the `if`/`in` on the next line right?